### PR TITLE
[docs] update doc links to quickstart/explore

### PR DESCRIPTION
**Pull request by polarweasel**

### CODE REVIEW

File: src/components/DockerInstall.vue [hunk 1]
```diff
@@ -64,7 +64,7 @@
       >
         <q-space />
         <div class="quickstart-container">
-          <a target="_blank" rel="noopener" id="macos-quickstart-link" href="https://docs.yugabyte.com/preview/quick-start/explore/ysql/#docker">Complete Docs</a>
+          <a target="_blank" rel="noopener" id="macos-quickstart-link" href="https://docs.yugabyte.com/preview/develop/build-apps/">Complete Docs</a>
```
Comment (ddhodge) [inline src/components/DockerInstall.vue:67]: Build apps? Not quite the same... there's this https://docs.yugabyte.com/preview/sample-data/retail-analytics/

File: src/components/AWSDeploy.vue [hunk 2]
```diff
@@ -37,7 +37,7 @@
       <div class="quickstart-container mobile-view">
         <a v-if="databaseTab === 'Terraform'" target="_blank" rel="noopener" id="macos-quickstart-link" href="https://github.com/yugabyte/terraform-aws-yugabyte">Complete Docs</a>
         <a v-else-if="databaseTab === 'EKS'" target="_blank" rel="noopener" id="macos-quickstart-link" href="https://docs.yugabyte.com/preview/deploy/kubernetes/multi-zone/eks/helm-chart/">Complete Docs</a>
-        <a v-else target="_blank" rel="noopener" id="macos-quickstart-link" href="https://docs.yugabyte.com/preview/deploy/public-clouds/aws/">Complete Docs</a>
+        <a v-else target="_blank" rel="noopener" id="macos-quickstart-link" href="https://docs.yugabyte.com/preview/deploy/public-clouds/aws/cloudformation/">Complete Docs</a>
```
Comment (ddhodge) [inline src/components/AWSDeploy.vue:40]: goes to cloudformation, but hovering over link don't see the change

**Review by ddhodge — COMMENTED** [review]
Deploy -> Google -> Cloud Deployment -> Complete Docs lands on Terraform page

File: src/components/AWSDeploy.vue (hunk 2 above)
Comment (aishwarya24) [inline src/components/AWSDeploy.vue:40]: Another observation related to [Terraform](https://deploy-preview-73--upbeat-panini-d4c778.netlify.app/cloud#aws) (I know you haven't changed this file in this PR , but wanted to point out anyway)- The _complete docs_ link for Terraform is pointing to its GitHub repo: https://github.com/yugabyte/terraform-aws-yugabyte and not to the docs link - https://docs.yugabyte.com/preview/deploy/public-clouds/aws/terraform/. This is true for AWS, GCP and Azure.

File: src/components/DockerInstall.vue (hunk 1 above)
Comment (polarweasel) [reply src/components/DockerInstall.vue:67]: changed

File: src/components/AWSDeploy.vue (hunk 2 above)
Comment (polarweasel) [reply src/components/AWSDeploy.vue:40]: thanks! i think i caught them all now...

Comment (polarweasel) [reply src/components/AWSDeploy.vue:40]: on the docs site, aws redirects to aws/cloudformation through an alias, but i've made the links explicit.

**Pull request merged**